src/index.html

```diff
@@ -1,13 +1,55 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link
+    href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+    rel="stylesheet"
+    >
+  </head>
+  <body>
+    <section
+     class="card card--position"
+     data-qa="card"
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="Product icon"
+        class="card__logo"
+      >
+      <p class="card__description">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+      <p class="card__code">
+        Product code: 195434
+      </p>
+      <div class="card__raiting">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="card__rating-number">Reviews: 5</p>
+      </div>
+      <div class="card__price">
+        <p class="card__price-text">Price:</p>
+        <p class="card__price-amount">$2,199</p>
+      </div>
+      <div class="card__button">
+        <a
+          href="#"
+          class="card__button-buy"
+          data-qa="hover"
+        >
+        BUY
```

Comment:
let's handle making this text uppercase with styles

Reply:
fixed